src/pages/learn/subscriptions.mdx

```diff
@@ -0,0 +1,102 @@
+# Subscriptions
+
+import { Callout } from "nextra/components"
+
+<p className="learn-subtitle">Learn how to get real-time updates from a GraphQL server</p>
+
+In addition to reading and writing data using stateless query and mutation operations, the GraphQL specification also describes how to fetch data in real-time via long-lived requests. On this page, we'll explore how clients can receive updated data in response to some event on a GraphQL server using subscription operations.
+
+<Callout type="info">
+Many of the features of GraphQL operations that apply to queries also apply to subscriptions, so review the [Queries](/learn/queries/) page first before proceeding.
+</Callout>
+
+## Subscribing to updates
+
+Subscription fields are defined exactly as `Query` and `Mutation` fields are, but using the `subscription` root operation type instead:
+
+```graphql
+type Subscription {
+  reviewCreated: Review
+}
+```
+
+Similarly, a subscription is initiated using the `subscription` keyword as the operation type:
+
+```graphql
+subscription NewReviewCreated {
+  reviewCreated {
+    rating
+    commentary
+  }
+}
+```
+
+GraphQL subscriptions are typically backed by a separate pub/sub system so that messages about updated data can be published as needed during runtime and then consumed by the resolver functions for the subscription fields in the API.
+
+For the previous example, we could imagine publishing a `REVIEW_CREATED` message to a pub/sub system when a client submits a new review with the `createReview` mutation, and then listening for this event in the resolver function for the `reviewCreated` subscription field so that the server may send this data to subscribed clients.
+
+<Callout type="info">
+Note that subscriptions are different from the concept of _live queries_, which are not formally described in the specification and are often implemented differently in GraphQL various implementations using an `@live` executable directive. [Read more about the ongoing discussion on a formal specification for live queries here.](https://github.com/graphql/graphql-spec/issues/386)
```

Comment:
Let's make it a little clearer that there's no standard implementation of live queries.

```suggestion
Note that subscriptions are different from the concept of _live queries_, a loosely defined feature that some GraphQL implementations may offer. [Read more about the ongoing discussion on a formal specification for live queries here.](https://github.com/graphql/graphql-spec/issues/386)
```